frontend/src/surveys/containers/GetSurveysList/GetSurveysList.js

```diff
@@ -0,0 +1,28 @@
+import { connect } from 'react-redux';
+
+import SurveysList from '../../components/SurveysList';
+import HttpUtility from '../../utils';
+// server.fetchDev(10);
+const getSurversFromServerSide = () => HttpUtility.GetSurveys();
```

Comment:
Why do you use HttpUtility in your components? You should dispatch redux actions